services/common/src/interfaces/projects/authorizationSummary.interface.ts

```diff
@@ -0,0 +1,7 @@
+export interface IAuthorizationSummaryColumn {
```

Comment:
I think it would make more sense to have the interface just be for "IAuthorizationSummary" (as it's the record type that the column is displaying and could be reused in other places with authorization summaries).